src/Examples/UnitModels/Reactors/plug_flow_reactor_testing.ipynb

```diff
@@ -0,0 +1,1147 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Flowsheet PFR Simulation and Optimization of Ethylene Glycol Production\n",
```

Comment:
Whilst most readers will probably understand, you should write Plug Flow Reactor out in full the first time you use it.